src/torrent.ts

```diff
@@ -201,8 +201,32 @@ export async function getTorrentByFuzzyName(
 	name: string
 ): Promise<null | Metafile> {
 	const allNames = await db("torrent").select("name", "file_path");
+
+	const episode = name.match(EP_REGEX);
+	const season = name.match(SEASON_REGEX);
+	const movie = name.match(MOVIE_REGEX);
+	const fullMatch = (episode?.[0] ?? season?.[0] ?? movie?.[0])
+		.replaceAll(".", "").replaceAll(" ", "").toLowerCase();
+
+	// Attempt to filter torrents in DB to match incoming torrent before fuzzy check
+	var filteredNames = [];
```

Comment:
nit: prefer `const` or `let` if mutable
```suggestion
	const filteredNames = [];
```